codes/dart/chapter_computational_complexity/space_complexity.dart

```diff
@@ -0,0 +1,109 @@
+/**
+ * File: space_complexity.dart
+ * Created Time: 2023-2-12
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+
+import 'dart:collection';
+import '../utils/list_node.dart';
+import '../utils/print_util.dart';
+import '../utils/tree_node.dart';
+
+class SpaceComplexity {
+  /* 函数 */
```

Comment:
Remove the class if it is unnecessary.

(In Java, we have to create a class but it is not elegant. You can refer to the C++ codes to see the function-only code.)